app/infrastructure/database/migrations/002_publications_create.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'sequel'
+
+Sequel.migration do
+  change do
+    create_table(:publications) do
+      primary_key :id
+      # 感覺錯錯的
+      # foreign_key  :publicationId, :papers
+      
+      Integer      :publicationId, unique: true, null: false
+      Float        :journalImpact, unique: false, null: false
+      Integer      :viewsCount, unique: false, null: false
+      Integer      :citationCount, unique: false, null: false
+      Integer      :publicationYear, unique: false, null: true
+      String       :sourceTitle, unique: false, null: true
```

Comment:
I would like to suggest you use snake case to make the column name since sqlite in ruby provide some funciton that can read the variable name with snake case. [FYR](https://en.wikipedia.org/wiki/Snake_case)